docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,685 @@
+---
+title: .NET Aspire 9.0 (Release Candidate 1)
+description: Learn what's new with .NET Aspire 9.0 (Release Candidate 1).
+ms.date: 10/11/2024
+---
+
+# .NET Aspire 9.0 (Release Candidate 1)
+
+.NET Aspire 9.0 is the next major release of .NET Aspire; it will support BOTH .NET 8 (LTS) and .NET 9. .NET Aspire 9.0 addresses some of the most highly requested features and pain points from the community. 
+
+## Upgrade to .NET Aspire 9.0
+
+Instructions on how to upgrade here
+
+- Visual Studio
+- Visual Studio Code
+- Command Line
+- Upgrade assistant ???
+
+### Acquisition
+
+.NET Aspire 9.0 simplifies acquisition by providing a .NET Aspire SDK available as the [📦 `Aspire.AppHost.Sdk`](https://nuget.org/packages/Aspire.AppHost.Sdk) NuGet package. This will no longer require an extra step to acquire the tooling required to build .NET Aspire projects. In other words, starting with .NET Aspire 9—you don't need to install a .NET workload.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Sdk Name="Aspire.AppHost.Sdk" Version="9.0.0-rc.1.24511.1" />
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsAspireHost>true</IsAspireHost>
+    <UserSecretsId>0afc20a6-cd99-4bf7-aae1-1359b0d45189</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Aspire.Hosting.AppHost" Version="9.0.0-rc.1.24511.1" />
+  </ItemGroup>
+
+</Project>
+```
+
+For more information, see [.NET Aspire app host project](../fundamentals/app-host-overview.md#app-host-project).
+
+### Templates
+
+.NET Aspire 9 introduces a new project template for creating .NET Aspire projects. This template is available from the [`dotnet new` command](/dotnet/core/tools/dotnet-new). These can be installed by running the following command:
+
+```dotnetcli
+dotnet new install Aspire.ProjectTemplates::9.0.0-rc.1.24511.1
```

Comment:
Seems like we are going to need to use ```--force``` with ```dotnet new install```. @joperezr just wanted to confirm this before I update this text.